languages/en.ini

```diff
@@ -746,7 +746,13 @@ load_tag_error = "Error: Could Not Load Tags"
 Loading = "Loading"
 loading_ellipsis = "Loading…"
 Loan History = "Loan History"
+loan_history_all_purged = "Loan history has been purged"
+loan_history_confirm_purge_all = "Do you wish to purge all loans from the loan history?"
```

Comment:
Would "your" loan history be better than "the" loan history (here and in other lines)?